metagraph/tests/annotation/test_converters.cpp

```diff
@@ -433,14 +444,16 @@ void test_row_diff_separate_columns(uint32_t k,
     graph->serialize(graph_fname);
 
     std::map<std::string, std::vector<uint64_t>> col_annotations;
-    for (uint32_t anno_idx = 0; anno_idx < graph->num_nodes(); ++anno_idx) {
+    uint32_t anno_idx = 0;
+    graph->call_nodes([&](auto node_idx) {
         for (const auto &label : annotations[anno_idx]) {
-            col_annotations[label].push_back(anno_idx);
+            col_annotations[label].push_back(node_idx - 1);
         }
-    }
+        ++anno_idx;
```

Comment:
do we still need this? why don't we just set `auto anno_idx = AnnotatedDBG::graph_to_anno_index(node_idx)` in the callback and work with that?